front/src/utils/request.js

```diff
@@ -11,10 +11,12 @@ export const post = (url, body, additionalConfig = {}) => fetch(url, {
 
 export const toJSON = (response) => response.json();
 
-export const get = (url) => fetch(url, {
+export const get = (url, additionalConfig = {}) => fetch(url, {
   method: 'GET',
   'credentials': 'include',
+  ...additionalConfig,
   headers: {
-    'Content-Type': 'application/json'
+    'Content-Type': 'application/json',
+    ...additionalConfig.headers
   }
-});
\ No newline at end of file
+});
```

Comment:
Very nice empty line

Reply:
i have done everything i can to fix that